mono/metadata/object.c

```diff
@@ -7197,6 +7228,24 @@ mono_object_handle_isinst_mbyref (MonoObjectHandle obj, MonoClass *klass, MonoEr
 
 	MonoObjectHandle result = MONO_HANDLE_NEW (MonoObject, NULL);
 
+	if (MONO_HANDLE_IS_NULL (obj))
+		goto leave;
+
+	gboolean success = mono_object_handle_isinst_mbyref_raw (obj, klass, error);
```

Comment:
Compiles for C++?

Reply:
Yeah, I saw the failure. And that's true, but I think that's a really wide `if` statement to put on one line.